modules/machineset-creating.adoc

```diff
@@ -18,194 +13,97 @@ permission.
-    machine.openshift.io/cluster-api-machine-type: <machine_label> <3>
-  name: <cluster_name>-<machine_label>-<AWS-availability-zone> <3> <4>
+    machine.openshift.io/cluster-api-cluster: <clusterID>
+  name: <clusterID>-<role>-us-east-1a
   namespace: openshift-machine-api
-  resourceVersion: "9249" <1>
-  selfLink: /apis/machine.openshift.io/v1beta1/namespaces/openshift-machine-api/machinesets/ <infrastructure_id>-<machine_label>-<AWS-availability-zone> <1>
-  uid: 59ba0425-313f-11e9-861e-0a18047f0a28 <1>
-----
-<1> Remove this line.
-<2> Do not change the `<infrastructure_id>`.
-<3> For each `<machine_label>` instance, specify the name of the new MachineSet.
-<4> Ensure that the AWS availability zone is correct in each instance of the
-`<AWS-availability-zone>` parameter.
-+
-The `metadata` section resembles the following YAML:
-+
-[source,yaml]
-----
-metadata:
-  labels:
-    machine.openshift.io/cluster-api-cluster: <infrastructure_id>
-    machine.openshift.io/cluster-api-machine-role: <new_machine_label>
-    machine.openshift.io/cluster-api-machine-type: <new_machine_label>
-  name: <cluster_name>-<new_machine_label>-<AWS-availability-zone>
-  namespace: openshift-machine-api
-----
-
-. In `<file_name>.yaml`, delete the `status` stanza:
-+
-[source,yaml]
-----
-status:
-  availableReplicas: 1
-  fullyLabeledReplicas: 1
-  observedGeneration: 1
-  readyReplicas: 1
-  replicas: 1
-----
-
-. In `<file_name>.yaml`, update both instances of the `machine.openshift.io/cluster-api-machineset` parameter
-values in the `spec` section to match the `name` that you defined in the `metadata` section:
-+
-[source,yaml]
-----
 spec:
   replicas: 1
   selector:
     matchLabels:
-      machine.openshift.io/cluster-api-cluster: <infrastructure_id>
-      machine.openshift.io/cluster-api-machineset: <cluster_name>-<machine_label>-<AWS-availability-zone> <1>
+      machine.openshift.io/cluster-api-cluster: <clusterID>
+      machine.openshift.io/cluster-api-machine-role: <role>
+      machine.openshift.io/cluster-api-machine-type: <role>
+      machine.openshift.io/cluster-api-machineset: <clusterID>-<role>-us-east-1a
   template:
     metadata:
-      creationTimestamp: null
       labels:
-        machine.openshift.io/cluster-api-cluster: <infrastructure_id>
-        machine.openshift.io/cluster-api-machine-role:  <machine_label>
-        machine.openshift.io/cluster-api-machine-type: <machine_label>
-        machine.openshift.io/cluster-api-machineset: <cluster_name>-<machine_label>-<AWS-availability-zone> <1>
-...
-----
-<1> Ensure that both the `machine.openshift.io/cluster-api-machineset` parameter values
-match the `name` that you defined in the `metadata` section.
+        machine.openshift.io/cluster-api-cluster: <clusterID>
+        machine.openshift.io/cluster-api-machine-role: <role>
+        machine.openshift.io/cluster-api-machine-type: <role>
+        machine.openshift.io/cluster-api-machineset: <clusterID>-worker-us-east-1a
```

Comment:
Should be `machine.openshift.io/cluster-api-machineset: <clusterID>-<role>-us-east-1a`